TWLight/emails/tasks.py

```diff
@@ -39,7 +39,6 @@
 from TWLight.emails.signals import ContactUs
 from TWLight.resources.models import AccessCode, Partner
 from TWLight.users.groups import get_restricted
-from TWLight.users.signals import Notice, ProjectPage2021Launch, ProxyBundleLaunch
```

Comment:
`Notice` and `ProjectPage2021Launch` are used for other signals, so you only need to remove `, ProxyBundleLaunch` here.